src/main/java/org/opentripplanner/routing/core/vehicle_sharing/BikePathVehicleDescription.java

```diff
@@ -0,0 +1,48 @@
+package org.opentripplanner.routing.core.vehicle_sharing;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.opentripplanner.routing.core.TraverseMode;
+
+public abstract class BikePathVehicleDescription extends VehicleDescription {
```

Comment:
może jakiś komentarz, że to są pojazdy, które mogą trasować się po chodniku i ścieżkach rowerowych